docs/.vuepress/site-config/sidebar-nav.js

```diff
@@ -1109,7 +1109,7 @@ module.exports = {
         "policies/proxy-template",
         "policies/general-notes-about-kuma-policies",
         "policies/how-kuma-chooses-the-right-policy-to-apply",
-        "policies/http-support-in-kuma",
+        "policies/supported-protocols",
```

Comment:
only the newest version should be changed

Reply:
okay